cmd-args.md

```diff
@@ -30,15 +30,15 @@ When you create a {{site.data.keyword.codeengineshort}} application or job, you
 | The arguments that are passed to the command.    | `CMD`    | `args` |
 {: caption="Docker and {{site.data.keyword.codeengineshort}} names" caption-side="bottom"}
 
-- If `--command` is used, then any image `Entrypoint` value is overwritten and any image `cmd` values are ignored.
-- If `--argument` is used, then any image `Command` value in overwritten.
+- If `--command` is used, then any image `Entrypoint` value is overwritten and any image `CMD` values are ignored.
```

Comment:
You are right, the current documentation mixes Terms and `cmd` is not ideal in this sentence. I suggest to stick to the terminology and use `Command` instead of `CMD`. Mainly because the we introduce these terms in the second paragraph and explain how they relate to the keywords used in the Dockerfile.

> Container images include two pieces of metadata that tell the container runtime what command, inside the image to run when the container is created. These metadata fields are called Entrypoint and Command. For those users who are familiar with Dockerfile, the fields equate to the ENTRYPOINT and CMD commands. These two fields contain arrays of strings that are combined to create the command line that is used when you run your container.

```suggestion
- If `--command` is used, then any image `Entrypoint` value is overwritten and any image `Command` values are ignored.
```

Reply:
i went back and forth on this myself. I'm happy to go either way, but if we go with `Command` i think we need to update the table as well to be completely consistent
|Description| Image name | Code Engine name|
|---|--------|-----|
| The command that is run by the container |`Entrypoint`|`command`|
| The arguments that are passed to the command |`Command`|`args`|